data/input_files/tests/solid/qs_linear.lua

```diff
@@ -39,11 +39,10 @@ solid = {
     -- neo-Hookean material parameters
     materials = { { model = "NeoHookean", mu = 0.25, K = 10.0, density = 1.0 }, },
 
-    -- Turn the geometric nonlinearities off
-    geometric_nonlin = false,
-
     -- Turn the material nonlinearities off
     -- TODO: this should be replaced with a proper material definition
+    -- BT 11/5/24: I think this option is outdated and no longer used, can an inlet
+    -- expert confirm?
     material_nonlin = false,
```

Comment:
there are no other uses of "material_nonlin" in the repo, so it's probably safe to remove?

Reply:
That's what I was thinking. @white238, I don't know this part well. Can you confirm?

Reply:
`geometric_nonlin` and `material_nonlin` where input options for the pre-functional `Solid` class. They are not used anymore and can be deleted.